.github/labeler.yml

```diff
@@ -0,0 +1,67 @@
+sig/engine:
+  - "cmd/*"
+  - "components/backup/*"
+  - "components/collections/*"
+  - "components/engine_*/*"
+  - "components/error_code/src/engine.rs"
+  - "components/error_code/src/lib.rs"
+  - "components/error_code/src/raftstore.rs"
+  - "components/error_code/src/sst_importer.rs"
+  - "components/error_code/src/storage.rs"
+  - "components/error_code/src/bin.rs"
```

Comment:
Maybe we can simply leave error code alone, since the code may be frequently changed.

Reply:
Got it.